src/pages/AdminDashboard/AdminSettings.tsx

```
@@ -19,14 +19,16 @@ const AdminSettings = () => {
-  const [message,setMessage] = useState("")
+ 
   const [showModal,setShowModal] = useState(false);
   const [changepassword,{isLoading}] = useChangePasswordMutation() as any
-  const [feedback] = useFeedbackMutation()
-  // const user_data: any = sessionStorage.getItem("userInfo");
-	// const userInfo = user_data && JSON.parse(user_data);
-  // const details = userInfo && userInfo?.data?.userDetails
+ 
+  const user_data: any = sessionStorage.getItem("userInfo");
+	const userInfo = user_data && JSON.parse(user_data);
+  const details = userInfo && userInfo.data?.data
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)